Makefile

```diff
@@ -1,7 +1,7 @@
 # SysMLv2 Release to use. First is release version of the API server,
 # the second is the release version of the SysMLv2
-release = 2024-07
-sysml_release= 2024-09
+release=2024-07
+sysml_release=2024-09
```

Comment:
Looks like we get to update again immediately.  See https://github.com/Systems-Modeling/SysML-v2-Release/releases/tag/2024-11

Reply:
Yup, I'd already seen the -11 release (it's the one in use on our shared server), I'll make that change before merging.

Reply:
Actually, I won't. I'll make another PR with that change.